.github/workflows/ci.yml

```
@@ -75,6 +75,15 @@ jobs:
       - name: Run E2E test with cache
         run: yarn workspace @ject-5-fe/app run test:e2e-cached
 
+      - name: build vercel preview with custom action
+        uses: ./.github/actions/vercel-deploy
```

actionlint (1.7.7)

invalid runner name "node22" at runs.using in "Custom Vercel Deploy" action defined at "/home/jailuser/git/.github/actions/vercel-deploy". valid runners are "composite", "docker", and "node20". see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs

(action)